RELEASE.md

```diff
@@ -0,0 +1,66 @@
+# LNC Release Process
+
+This document describes the steps needed to release a new version of LNC binaries.
+
+### System Requirements
+
+1. Android Studio with Android SDK (API level 16 or newer)
+2. Xcode (latest version)
+3. Go v1.19.8 or newer
+
+### Build Release Binaries
+
+From the root of the project, run the following command. Replace `vX.Y.Z-alpha`
+with the next version number (ex: v0.2.0-alpha)
+
+```sh
+$ make release version=vX.Y.Z-alpha
```

Comment:
Nice! Just one last request: Can we call the make variable `tag` instead of `version`? Then it would be the same in most of our repos.

Reply:
Easy enough. I changed it to `tag`.